scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/SoftKeywords.scala

```diff
@@ -0,0 +1,87 @@
+package scala.meta.internal.parsers
+
+import scala.meta.Dialect
+import scala.meta.tokens.Token
+import scala.meta.tokens.Token.Ident
+import scala.meta.internal.classifiers.classifier
+
+class SoftKeywords(dialect: Dialect) {
+
+  def isIdentAnd(token: Token, pred: String => Boolean): Boolean = token match {
+    case Ident(value) if pred(value.stripPrefix("`").stripSuffix("`")) => true
+    case _ => false
+  }
+
+  def isSoftKw(token: Token, name: String): Boolean = isIdentAnd(token, _ == name)
+
+  @classifier
+  trait SkAs {
```

Comment:
Nitpick: I'm not a fan of the "Sk" prefix. Instead of using wildcard imports, how do you like this syntax?
```suggestion
class SoftKeywords {
  trait KwAs { ... }
}

val soft = new SoftKeywords()
x match {
  case soft.KwAs
```

Reply:
Looks good, I will change it!

Reply:
Reafctored!